crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -116,6 +116,15 @@ def repocheck
       end
     end
 
+    def compute_resources_available?
+      compute_nodes = NodeObject.find("roles:nova-compute-kvm")
```

Comment:
Hm, good point. We should probably do also a search for xen compute roles. However, I assume it's not possible to migrate instances from kvm host to xen one (and vice versa) so it should be additional check.
(Correct me if I'm wrong here)